### PR TITLE
AGM-1107 removed some info

### PR DESCRIPTION
**Pull request by anastasiia-fedotova**

### CODE REVIEW

File: 11.vpn/010.adguard-vpn-general/020.adguard-vpn-is-the-better-option/docs.en.md [hunk 1]
```diff
@@ -70,10 +70,11 @@ This feature has been developed specifically for unsecured networks. For the con
...
 AdGuard VPN offers five simultaneous connections to its paid users. We place special emphasis on the word ’*simultaneous*’, and here is why. Some VPN providers only allow to log in to their service on *five devices max*. With AdGuard VPN, *there’s no limit* to the number of devices that can be logged in to your account, but you can have five VPN connections enabled at the same time. Those who don’t have a subscription can use AdGuard VPN on *two devices*, which gives us another trump card to play against competitors — most of the VPN services only have *one device* in their free versions.
 
-## 11. Streaming support  
-According to statistics, about 20% of VPN users install it primarily to binge-watch shows, series and films on streaming services. The problem is it’s not easy to outsmart streaming services as they can detect VPNs and block sneaky users access to their platform.
+## 11. Secure streaming 
+Streaming services don't like VPNs for obvious reasons: according to statistics, about 20% of users install a VPN it mainly to watch shows, TV shows and movies bypassing geo-blocking. That's why streaming platforms tend to do everything they can to track VPN traffic and block it.
```
Comment (vbagirov) [inline 11.vpn/010.adguard-vpn-general/020.adguard-vpn-is-the-better-option/docs.en.md:74]: я думаю проще этот пункт вообще убрать

**Review by vbagirov — CHANGES_REQUESTED** [review]

File: 11.vpn/010.adguard-vpn-general/020.adguard-vpn-is-the-better-option/docs.en.md (hunk 1 above)
Comment (anastasiia-fedotova) [reply 11.vpn/010.adguard-vpn-general/020.adguard-vpn-is-the-better-option/docs.en.md:74]: Дэн одобрил такой вариант (без перечисления сервисов):

## 11. Secure streaming

Streaming services don't like VPNs for obvious reasons: according to statistics, about 20% of users install a VPN mainly to watch shows, TV shows and movies bypassing geo-blocking. That's why streaming platforms tend to do everything they can to track VPN traffic and block it.

But what if you want to watch content specific to your region while feeling safe? Or don't want stop watchig exciting series even when you travel to another country?The answer is simple — AdGuard VPN.

Comment (vbagirov) [reply 11.vpn/010.adguard-vpn-general/020.adguard-vpn-is-the-better-option/docs.en.md:74]: Ну надо тогда подчеркнуть что типа стриминги блочат впны, но с адичем типа всё збс, только не используйте его в нелегальных целях

**Review by vbagirov — CHANGES_REQUESTED** [review]

File: 11.vpn/010.adguard-vpn-general/020.adguard-vpn-is-the-better-option/docs.en.md (hunk 1 above)
Comment (anastasiia-fedotova) [reply 11.vpn/010.adguard-vpn-general/020.adguard-vpn-is-the-better-option/docs.en.md:74]: But what if you want to watch content specific to your region while feeling safe? Or don't want stop watchig exciting series even when you travel to another country? The answer is simple — AdGuard VPN, which thanks to its unique protocol can remain invisible to services. 

> Before you start watching a show on a streaming service with a VPN enabled, make sure you have a subscription and your actions are legal.

**Review by vbagirov — APPROVED** [review]

**Pull request merged**